.gitmodules

```diff
@@ -293,3 +293,6 @@
 	path = third_party/imgui/repo
 	url = https://github.com/ocornut/imgui
 	platforms = linux
+[submodule "third_party/infineon/trustm/optiga-trust-m"]
+	path = third_party/infineon/trustm/optiga-trust-m
+	url = https://github.com/Infineon/optiga-trust-m
```

Comment:
Shouldn't this be limited to some set of platforms?

Reply:
Hi @bzbarsky-apple, Thanks for your review. We have added the platform in the latest commit.